x-pack/plugins/security_solution/server/endpoint/endpoint_app_context_services.ts

```diff
@@ -118,6 +120,18 @@ export class EndpointAppContextService {
         getPackagePolicyDeleteCallback(dependencies.exceptionListsClient)
       );
     }
+
+    if (this.startDependencies.registerListsServerExtension) {
+      const { registerListsServerExtension } = this.startDependencies;
+
+      registerListsServerExtension({
+        type: 'exceptionsListPreCreateItem',
+        callback: async (arg) => {
+          this.startDependencies?.logger.info('exceptionsListPreCreateItem called!');
```

Comment:
I will remove this `logger`, but will leave the rest in place even if not doing anything at this point. We are going to have subsequent PRs on the security solution side to add logic into these two new extension points.